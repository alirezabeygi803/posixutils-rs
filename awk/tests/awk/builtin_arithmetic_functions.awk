BEGIN {
    print atan2(1, 1);
    print cos(0);
    print exp(1);
    print int(1.5);
    print log(1);
    print sqrt(4);
}