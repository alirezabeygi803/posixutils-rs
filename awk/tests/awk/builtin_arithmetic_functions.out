0.785398
1
2.71828
1
0
2
